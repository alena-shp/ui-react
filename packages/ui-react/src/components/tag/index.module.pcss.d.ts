export const tag: string
export const tag_interactive: string


interface Stylesheet {
    'tag': string
    'tag_interactive': string
    'tag_size-large': string
    'tag_size-medium': string
    'tag_style-badge-active': string
    'tag_style-badge-failed': string
    'tag_style-badge-processing': string
    'tag_style-badge-purple': string
    'tag_style-badge-rollbacked': string
    'tag_style-badge-success': string
}


declare const stylesheet: Stylesheet
export default stylesheet
