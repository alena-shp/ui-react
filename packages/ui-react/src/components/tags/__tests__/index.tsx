import React from 'react';
import { create } from 'react-test-renderer';

import { Tags } from '..';
import { Tag } from '../../tag';

test('Default', () => {
    const tree = create(
        <Tags>
            <Tag children="Название тега" />
            <Tag children="Название тега #2" />
            <Tag children="Название тега #3" />
        </Tags>
    );

    expect(tree.toJSON()).toMatchSnapshot();
});

test('Custom tagName', () => {
    const tree = create(
        <Tags tagName="ul">
            <Tag children="Название тега" />
            <Tag children="Название тега #2" />
            <Tag children="Название тега #3" />
        </Tags>
    );

    expect(tree.toJSON()).toMatchSnapshot();
});
