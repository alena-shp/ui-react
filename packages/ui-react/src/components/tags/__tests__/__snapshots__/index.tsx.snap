// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom tagName 1`] = `
<ul
  className="tags"
>
  <span
    className="tag tag_style-tag tag_size-small"
  >
    Название тега
  </span>
  <span
    className="tag tag_style-tag tag_size-small"
  >
    Название тега #2
  </span>
  <span
    className="tag tag_style-tag tag_size-small"
  >
    Название тега #3
  </span>
</ul>
`;

exports[`Default 1`] = `
<div
  className="tags"
>
  <span
    className="tag tag_style-tag tag_size-small"
  >
    Название тега
  </span>
  <span
    className="tag tag_style-tag tag_size-small"
  >
    Название тега #2
  </span>
  <span
    className="tag tag_style-tag tag_size-small"
  >
    Название тега #3
  </span>
</div>
`;
